Source/Libraries/CorruptCore/BlastRadius.cs

```diff
@@ -0,0 +1,267 @@
+namespace RTCV.CorruptCore
+{
+    using System;
+    using System.Linq;
+    using System.Windows.Forms;
+    using RTCV.Common.CustomExtensions;
+    using RTCV.NetCore;
+    using BlastRadiusAlgorithm = System.Func<RTCV.CorruptCore.BlastInfo, RTCV.CorruptCore.BlastLayer>;
+
+    public static class BlastRadius
+    {
+        public static BlastRadiusAlgorithm GetFromString(string str)
```

Comment:
I'm not happy with this `BlastRadiusAlgorithm => string` and `string => BlastRadiusAlgorithm` code. Fine for v1 but I'm open to a better solution.